api/XMLHttpRequest.json

```diff
@@ -1015,7 +1021,7 @@
                 "version_added": "31"
               },
               "chrome_android": {
-                "version_added": true
+                "version_added": "55"
```

Comment:
What data is this mirroring? I would expect "31" here.

Reply:
This was to fix a consistency issue, but another merged PR already updates this data and sets it to "31", which seems more reasonable!